metrics/internal/collectors/storageconsumer.go

```diff
@@ -162,3 +162,16 @@ func getAllStorageConsumers(lister StorageConsumerLister) (storageConsumers []*o
 	}
 	return
 }
+
+func (c *StorageConsumerCollector) GetClientId() string {
+	storageConsumerLister := NewStorageConsumerLister(c.Informer.GetIndexer())
+	storageConsumers := getAllStorageConsumers(storageConsumerLister)
+	for _, storageConsumer := range storageConsumers {
+		// getting consumer id from the internal storageconsumer
```

Comment:
as per discussion from last meeting, it is better to get uid from storageconsumer CR then the client id, hence added the change
cc @Madhu-1 @nb-ohad